docs/layouts/shortcodes/generated/dynamic_section.html

```diff
@@ -80,6 +80,12 @@
             <td>Boolean</td>
             <td>Enables last-state fallback for savepoint upgrade mode. When the job is not running thus savepoint cannot be triggered but HA metadata is available for last state restore the operator can initiate the upgrade process when the flag is enabled.</td>
         </tr>
+        <tr>
+            <td><h5>kubernetes.operator.job.upgrade.last-state.checkpoint.max.age</h5></td>
```

Comment:
`kubernetes.operator.job.upgrade.last-state.max.allowed.checkpoint.age` reflects better the intention